src/App.jsx

```diff
@@ -53,14 +42,12 @@ function App() {
           </Modal>
           )}
 
-        {showVideoDetails
-          ? <VideoDetails video={videoDetail} onVideoDetailClose={hideVideoDetails} />
-          : <Header onAddClick={modal.showAdd} />}
-        <Body
-          showEditVideoModal={modal.showEdit}
-          showDeleteModal={modal.showConfirm}
-          onVideoDetailClick={handleVideoClick}
-        />
+        <Routes>
+          <Route path="/" element={<Navigate to="/search" replace />} />
```

Comment:
NIT: would recommend to keep all url constants in single constant file, so it will be a single source of truth for re-usage allover the applicaiton

you can also have util function to construct proper url for /search cases with all necessary parameters.. But this is not mandatory